src/pmi/src/pmix.c

```diff
@@ -501,7 +501,7 @@ static char *value_to_wire(pmix_value_t * val)
         PMIU_Assert(rc == MPL_SUCCESS);
     }
 
-    return encoded_value;
+    return wire;
```

Comment:
I think it is better to fix the "overwritten" behavior. E.g. use a temp var for the data part.

Reply:
Done.